tests/acceptance/reminder-list-test.js

```diff
@@ -158,3 +158,25 @@ test('clicking the Undo button in the edit form reverts to the original', functi
     assert.equal(find('.spec-reminder-item:first').text().trim(), 'Megatron');
   });
 });
+
+test('User is alerted that they have unsaved changes when view the edit form', function(assert){
+  visit('/');
+  click('.spec-add-new-form');
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/new');
+    fillIn('.new-reminder-title', 'Blue Whale');
+    click('.spec-add-new');
+    click('.spec-reminder-item:first');
+  });
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/1');
+    click('.spec-edit-reminder');
+  });
+
+  andThen(function(){
+    assert.equal(currentURL(),'/reminders/1/edit');
```

Comment:
See above comment but the `spec-edit-status` class should only be visible if a model has dirty attributes